lib/chromic_pdf/pdf/protocols/blank.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ChromicPDF.Blank do
```

Comment:
```suggestion
defmodule ChromicPDF.Reset do
```
like in `CSS` :P

or maybe `ResetTarget` ?